src/seedu/addressbook/Main.java

```diff
@@ -124,6 +128,14 @@ private StorageFile initializeStorage(String[] launchArgs) throws InvalidStorage
         boolean isStorageFileSpecifiedByUser = launchArgs.length > 0;
         return isStorageFileSpecifiedByUser ? new StorageFile(launchArgs[0]) : new StorageFile();
     }
-
+    
+    private void printTagsHistory() {
+    	UniquePersonList persons = addressBook.getAllPersons();
+    	for (Person person : persons) {
+    		for (Tagging tagHistory : person.getTagsHistory()) {
+    			ui.showToUser(tagHistory.toString());
```

Comment:
Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!